From 9080f619461f3d68c8126dcde7c6f95dbae75347 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 7 Jun 2023 13:55:21 +0300
Subject: [PATCH] [COFF] Remove misleading and unclear comments. NFC.

It is not planned that GNU binutils would change this aspect of
its behaviour wrt how symbol decoration is done.

Differential Revision: https://reviews.llvm.org/D152359
---
 include/llvm/Object/COFFModuleDefinition.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/llvm/Object/COFFModuleDefinition.h b/include/llvm/Object/COFFModuleDefinition.h
index 8e14dd614..c9d8f4289 100644
--- a/include/llvm/Object/COFFModuleDefinition.h
+++ b/include/llvm/Object/COFFModuleDefinition.h
@@ -39,9 +39,6 @@ struct COFFModuleDefinition {
   uint32_t MinorOSVersion = 0;
 };
 
-// mingw and wine def files do not mangle _ for x86 which
-// is a consequence of legacy binutils' dlltool functionality.
-// This MingwDef flag should be removed once mingw stops this pratice.
 Expected<COFFModuleDefinition>
 parseCOFFModuleDefinition(MemoryBufferRef MB, COFF::MachineTypes Machine,
                           bool MingwDef = false);
-- 
2.41.0.windows.1

